crates/bevy_ui/src/widget/label.rs

```diff
@@ -0,0 +1,9 @@
+use bevy_ecs::prelude::Component;
+use bevy_ecs::reflect::ReflectComponent;
+use bevy_reflect::std_traits::ReflectDefault;
+use bevy_reflect::Reflect;
+
+/// Marker struct for labels
```

Comment:
Nit: It might be good to indicate why this component should be used. Seems like it's mainly for the `Role::Label`, but that's not immediately obvious (users may skip this in favor of just relying on `Text`).

Reply:
If you have any wording suggestions I'm open to them--brain is a bit fried ATM. Alternately I can just remove it in favor of attaching a custom `NodeBuilder` to those nodes in particular. I added `Label` before I realized how easy it was to just slap on a custom `NodeBuilder` and make whatever customizations I needed there.

Reply:
Also, I feel like a label is a fairly common UI widget type, and we're going to want this or something like it at some point anyway. So it isn't really an accessibility thing, more a general widget I probably introduced before it's time thing.

Reply:
I don't have any wording suggestions off the top of my head. If it's meant to be part of a larger widget ecosystem, then maybe we come back to this (proper documentation) when we're ready to fully implement it? I imagine its usage and design could change a lot between now and when we truly define a "label" widget.